### PR TITLE
allow multiple runners/schemes

### PR DESCRIPTION
**Pull request by kbrock**
built upon:
- [ ] #152 (first two commits)

this adds scheme to `set_docker_runner`

### CODE REVIEW

File: lib/floe/workflow/runner.rb [hunk 1]
```diff
@@ -5,29 +5,50 @@ class Workflow
...
+              Floe::Workflow::Runner::Kubernetes.new(options)
+            when Floe::Workflow::Runner
+              name_or_instance
+            else
+              raise ArgumentError, "docker runner must be one of: docker, podman, kubernetes"
+            end
+        end
+
+        def docker_runner(scheme)
+          set_runner if @runners.empty?
```
Comment (Fryguy) [inline lib/floe/workflow/runner.rb:43]: Shouldn't the scheme be passed along?

```suggestion
        def docker_runner(scheme)
          set_runner(schema) if @runners.empty?
```

Comment (kbrock) [reply lib/floe/workflow/runner.rb:38]: This sets the default runner.

**Comment by kbrock** [comment]
update:
- scheme is no longer optional
- no longer have default values for scheme/instance name.
- rename some cases of schema to scheme

update:
- fix `exe/floe` to pass scheme along
- fix refactor bug where scheme was not created correctly

**Comment by kbrock** [comment]
merged into #152

**Pull request closed without merging**